cluster_config/configurations.yaml.tpl

```diff
@@ -2,48 +2,38 @@
-    "spark.executor.cores": "5"
+    "spark.executor.cores": "1"
     "spark.executor.extraJavaOptions": "-verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=70 -XX:MaxHeapFreeRatio=70 -XX:+CMSClassUnloadingEnabled -XX:OnOutOfMemoryError='kill -9 %p' -Dhttp.proxyHost='${proxy_http_address}' -Dhttp.proxyPort='3128' -Dhttp.nonProxyHosts='${proxy_no_proxy}' -Dhttps.proxyHost='${proxy_http_address}' -Dhttps.proxyPort='3128'"
     "spark.driver.extraJavaOptions": "-XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=70 -XX:MaxHeapFreeRatio=70 -XX:+CMSClassUnloadingEnabled -XX:OnOutOfMemoryError='kill -9 %p' -Dhttp.proxyHost='${proxy_http_address}' -Dhttp.proxyPort='3128' -Dhttp.nonProxyHosts='${proxy_no_proxy}' -Dhttps.proxyHost='${proxy_http_address}' -Dhttps.proxyPort='3128'"
     "spark.sql.warehouse.dir": "s3://${s3_published_bucket}/analytical-dataset/hive/external"
-    "spark.executor.memoryOverhead": "1200M"
-    "spark.driver.memory": "11000M"
-    "spark.executor.instances": "5"
-    "spark.dynamicAllocation.enabled": "false"
-    "spark.default.parallelism": "30"
+    "spark.dynamicAllocation.enabled": "true"
+    "spark.default.parallelism": "30",
+    "spark.dynamicAllocation.initialExecutors":"10",
+    "spark.dynamicAllocation.minExecutors":"10",
+    "spark.dynamicAllocation.maxExecutors":"20"
 
 - Classification: "spark-hive-site"
   Properties:
-    "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory"
+    "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory",
+    "hbase.zookeeper.quorum": "${zookeeper_quorum}"
 - Classification: "hive-site"
   Properties:
-    "hive.metastore.schema.verification": "false"
-    "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory"
-    "hive.metastore.warehouse.dir": "s3://${s3_published_bucket}/analytical-dataset/hive/external"
-- Classification: "hbase-site"
-  Properties:
-    "hbase.rootdir": "${hbase_root_path}"
-- Classification: "hbase"
-  Properties:
-    "hbase.emr.storageMode": "s3"
-    "hbase.emr.readreplica.enabled": "true"
+    "hive.metastore.schema.verification": "false",
+    "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory",
+    "hive.metastore.warehouse.dir": "s3://${s3_published_bucket}/analytical-dataset/hive/external",
+    "hbase.zookeeper.quorum": "${zookeeper_quorum}"
 - Classification: "emrfs-site"
```

Comment:
We can get rid of emrfs-site now too :)

Reply:
yeah :). will do it . Thanks Matt

Reply:
Matt, we can't - we're still writing to S3, just not via HBase!